README.md

```diff
@@ -31,82 +69,134 @@ Add this plugin like this:
 
 ## Example usage
 
-Let's start by creating a **root meta feed**, necessary for using this module.
-It lives alongside your existing "classical" feed, which we'll refer to as
-**main feed**.
-
+We create a subfeed for messages which describe us under our `root` feed
 ```js
-sbot.metafeeds.findOrCreate((err, metafeed) => {
-  console.log(metafeed) // { seed, keys }
+const find = (feed) => (
+  feed.feedpurpose === 'aboutMe' &&
+  feed.feedformat === 'classic'
+)
+const details = {
+  feedpurpose: 'aboutMe',
+  feedformat: 'classic',
+}
+
+sbot.metafeeds.findOrCreate(null, find, details, (err, aboutMeFeed) => {
```

Comment:
This first `null` argument may be confusing for beginners, and it's not explained that `null` is actually a short for "root" feed. I think you could restore the old example of first creating a root metafeed and then explicitly passing that root metafeed to this first argument, when creating the subfeed. This way readme readers can understand that we're "finding something" under that root metafeed, and then "creating under that root metafeed" in case it doesn't exist.

Reply:
You might be right.
I was trying to think "what is the most common use case of this".
And this is it (I think), but you're right it doesn't aid in learning in the same way.

Will add back in